apps/AEPSampleApp/extensions/Consent.js

```diff
@@ -13,25 +13,55 @@ governing permissions and limitations under the License.
 @format
 */
 
-import React, {Component} from 'react';
+import React, {useState, Component} from 'react';
 import {Button, StyleSheet, Text, View, ScrollView} from 'react-native';
 import {AEPConsent} from '@adobe/react-native-aepedgeconsent';
+import {AEPCore} from '@adobe/react-native-aepcore';
 
 export default Consent = ({ navigation }) => {
+  const [version, setVersion] = useState('');
+  const [consents, setConsents] = useState('');
+  AEPConsent.extensionVersion().then(version => setVersion(version));
+
+  function getConsents() {
+    var consents  = {"consents" : {"collect" : {"val": "n"}}};
+    AEPConsent.getConsents().then(currentConsents => {
+      let consentsStr = JSON.stringify(currentConsents);
+      setConsents(consentsStr);
+      console.log("AdobeExperienceSDK: AEPConsent.getConsents returned current consent preferences:  " + consentsStr);
+    }).catch((error) => {
+      console.warn("AdobeExperienceSDK: AEPConsent.getConsents returned error: ", error);
+    });
+  }
 
   return (
     <View style={styles.container}>
         <ScrollView contentContainerStyle={{ marginTop: 75 }}>
         <Button onPress={() => navigation.goBack()} title="Go to main page" />
-        <Text style={styles.welcome}>Consent</Text>
-        <Button title="extensionVersion()" onPress={consentExtensionVersion}/>
+        <Text style={styles.welcome}>Consent v{version}</Text>
+        <Button title="Set Collect Consent - Yes" onPress={() => updateCollectConsent(true)}/>
+        <Button title="Set Collect Consent - No" onPress={() => updateCollectConsent(false)}/>
+        <Button title="Set Default Consent - Yes" onPress={() => setDefaultConsent(true)}/>
```

Comment:
I have not change them so they match better with {"val": "y"} -> Yes, Allowed is just the name of the param used in the sample app

Reply:
Ok. but the output can be "collect":{"val":"n"} if I click "Set Collect Consent - No", then "Set  Default Consent - Yes", right?
The Yes is confusing in this case.

Reply:
The output is No because it was set to No previously and the default consent only influences the initial default consent settings. Once it was first set using the update API the default value is overwritten. Regardless of that if you click Set Default Consent - Yes you should see an update configuration with "collect": {"val":"y"}

Reply:
Got it. Then we can have SET DEFAULT CONSENT - YES placed to the top?   So user can check it after the launch?  I was clicking the SET COLLECT CONSENT buttons first,  so I couldn't get the value for DEFAULT button properly.

Reply:
Makes sense and thanks for the feedback, I updated.